spec/rubocop/cop/lint/unneeded_disable_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '.check' do
     let(:cop) do
       cop = described_class.new
-      cop.instance_eval { @options[:auto_correct] = true }
+      cop.instance_variable_get(:@options)[:auto_correct] = true
```

Comment:
I have nothing wrong with the test change here, but from a design perspective is it possible to set auto_correct in a less invasive way?

Reply:
I tried a few different ways, but couldn't find any. 😕 Will need to look into the implementation, and why this cop is seemingly a special case.